### DISCUSSION TITLE
osu website not a part of osu?

### DISCUSSION THREAD

**Discussion by jojopipe** [ppy/osu#20430] [Q&A] [answered]
### Type

Cosmetic

### Bug description

![image](https://user-images.githubusercontent.com/78083515/191972572-1eb0fd9e-8864-4f96-803d-49df752f651a.png)
^^^  when clicking the osu supporter logo on your little profile card ^^^

ok, so this could be just me, but this warning shouldn't really be there I feel like. I mean it tells you that you are 'leaving osu', but that's not really the case, is it?

### Version

2022.523.0 (newest)

### Logs

[performance.log](https://github.com/ppy/osu/files/9634091/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/9634092/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/9634093/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/9634094/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/9634096/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/9634097/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9634098/updater.log)
[database.log](https://github.com/ppy/osu/files/9634099/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/9634100/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/9634101/network.log)

not really necessary for this but sure, have my logs :)

**Comment by peppy** [comment, marked as answer]
This was an intentional choice as it still loses focus to the game.

Most pieces of the website are implemented natively in the game, and I think it still makes sense to warn users before the game is going to be minimised to show a web browser.
